### ISSUE TITLE
Submit PAH VCEP VCI records with new Batch API for ClinVar

### ISSUE THREAD

**Issue by larrybabb** [clingen-data-model/clinvar-api#8]
From SHarrison on 4/1/21 at 12:12p ET
----
Larry and Kyle: the PAH VCEP just sent me a submission of 40 variants that you could use as a test? I believe the PAH VCEP is one that we have already done an API submission on so we should already have the API key...?

From MLandrum (ClinVar) 
----
[Updated ClinVar API Document](https://app.zenhub.com/files/313686680/819e1dbf-b1b2-4668-8907-eafd051a350a/download)
Hi Larry, Steven, and Kyle,

We've updated the ClinVar submission API so that you can submit a batch of records now, not just a single record at a time. The documentation has also been updated; please see the attachment and let us know if anything in the documentation isn't clear. If you have a batch of data that's ready to submit, we'd love for you to try it out and let us know how it works for you.

We hope you find this updated version to be useful. Thanks so much for being early adopters!

Melissa

**Comment by larrybabb** [comment]
@theferrit32  Waiting on Steven Harrison to share the PAH VCEP submission file so that we can try this.
Once we have it we will review the new batch api and make adjustments to utilize the new feature.
When that is prepped, we will notify ClinVar & SH via email that we are about to send it
Then we will share the outcome via email with the same group.

**Comment by theferrit32** [comment]
Submission is complete and code is merged. We might want another ticket to validate the submitted records are in the latest clinvar xml archive
